kani-driver/src/concrete_playback/test_generator.rs

```diff
@@ -231,8 +235,51 @@ impl KaniSession {
     }
 }
 
+fn gen_test_doc(harness: &HarnessMetadata, property: &Property) -> String {
+    let mut doc_str = match &harness.attributes.kind {
+        HarnessKind::Proof => {
+            format!("/// Test generated for harness `{}` \n", harness.pretty_name)
+        }
+        HarnessKind::ProofForContract { target_fn } => {
+            format!(
+                "/// Test generated for harness `{}` that checks contract for `{target_fn}`\n",
+                harness.pretty_name
+            )
+        }
+        HarnessKind::Test => {
+            unreachable!("Concrete playback for tests is not supported")
+        }
+    };
+    doc_str.push_str("///\n");
+    doc_str.push_str(&format!(
+        "/// Check for `{}`: \"{}\"\n",
+        property.property_class(),
+        property.description
+    ));
+    if !harness.attributes.stubs.is_empty() {
+        doc_str.push_str("///\n");
+        doc_str.push_str(
+            "/// # Warning\n\
+               ///\n\
+               /// Concrete playback tests combined with stubs or contracts is highly \n\
+               /// experimental, and subject to change.\n\
+               ///\n\
+               /// The original harness has stubs which are not applied to this test.\n\
+               /// This may cause a mismatch of non-deterministic values if the stub\n\
+               /// creates any non-deterministic value.\n\
+               /// The execution path may also differ, which can be used to refine the stub\n\
+               /// logic.",
+        );
```

Comment:
```suggestion
doc_str.push_str(r#"/// 
/// # Warning
/// 
/// Concrete playback tests combined with stubs or contracts is highly 
/// experimental, and subject to change.
/// 
/// The original harness has stubs which are not applied to this test.
/// This may cause a mismatch of non-deterministic values if the stub
/// creates any non-deterministic value.
/// The execution path may also differ, which can be used to refine the stub
/// logic.
"#);
```